api-examples/get_log4j/get_log4j.py

```diff
@@ -0,0 +1,147 @@
+# !/usr/bin/env python3
+import json
+import requests
+import os
+from pprint import pprint
+from pprint import pformat
+
+try:
+    SHIFTLEFT_ORG_ID = os.environ["SHIFTLEFT_ORG_ID"]
+    SHIFTLEFT_ACCESS_TOKEN = os.environ["SHIFTLEFT_ACCESS_TOKEN"]
+except KeyError:
+    raise SystemExit("Oops! Do not forget to set both SHIFTLEFT_ORG_ID and SHIFTLEFT_ACCESS_TOKEN!")
```

Comment:
Do not raise, try 

```python
print("Oops! Do not forget to set both SHIFTLEFT_ORG_ID and SHIFTLEFT_ACCESS_TOKEN!")
os.Exit(1)
```
raise will print a traceback, actually this whole block should not be a try except

```python
SHIFTLEFT_ORG_ID = os.environ.get("SHIFTLEFT_ORG_ID", None)
SHIFTLEFT_ACCESS_TOKEN = os.environ.get("SHIFTLEFT_ACCESS_TOKEN", None)

if SHIFTLEFT_ORG_ID is None or SHIFTLEFT_ACCESS_TOKEN is None:
    print("Oops! Do not forget to set both SHIFTLEFT_ORG_ID and SHIFTLEFT_ACCESS_TOKEN!")
    os.Exit(1)